lib/routes/account.js

```diff
@@ -314,6 +315,7 @@ module.exports = function (
             uid: isA.string().regex(HEX_STRING).required(),
             sessionToken: isA.string().regex(HEX_STRING).required(),
             keyFetchToken: isA.string().regex(HEX_STRING).optional(),
+            challenge: isA.string().optional(),
```

Comment:
@vbudhram - I propose the scope if this particular PR to be reduced. Can we have this PR only deal with adding `mailer.sendConfirmLoginEmail` so that we can merge more quickly while other details are still being worked out?